src/aerie_cli/aerie_host.py

```diff
@@ -260,7 +264,15 @@ def is_auth_enabled(self) -> bool:
 
         return True
 
-    def authenticate(self, username: str, password: str = None):
+    def authenticate(self, username: str, password: str = None, override: bool = False):
+
+        try:
+            self.check_aerie_version()
+        except AerieHostVersionError as e:
+            if override:
+                print("Warning: " + e.args[0])
+            else:
+                raise
```

Comment:
Shouldn't this say `raise e`?

Reply:
Having `raise` in an `except` just re-raises the last exception that was caught, so it should be equivalent. The unit tests exercise this

Reply:
Understood! I was guessing it either did that or raised a generic Exception, thanks for confirming!